readme.md

```diff
@@ -191,24 +194,88 @@ Print is the only language defined i/o function. It takes in a single variable a
 The following are valid print statements:
 ```
 print ( "hello" );
-print ( a );
+print ( 42 );
+print ( a );  # Only valid if the variable "a" is defined.
 ```
 
 ### Loops
 A loop will itterate between a range of numbers.
+
 Example loops:
 ```
-loop ( a in range, ( 1, 100 ) ) {
+loop ( a in range( 1, 100 ) ) {
 
 	print ( a );
 
 }
 ```
+### Type casting
+Most data types can be converted into any other data type.
+```
+# Convert to int.
+# Can be done with all data types.
+int(x)
+
+# Convert to float.
+# Can be done with all data types.
+float(x)
+
+# Convert to string.
+# Can be done with all data types.
+string(x)
+
+# Convert to bool.
+# Can be done with all data types.
+bool()
+
+# Convert to none.
+# Can be done with all data types.
+none(x)
+```
+
+### NOP
```

Comment:
Is NOP needed? A empty set of braces is a NOP by design, you don't need a keyword for it.

Reply:
Good point, it isn't needed.

Reply:
I like the idea of pass, I believe it was floating around in my notes during the initial design but didn’t make it into the spec